Jitutils build is failing, wonder if 6.0 packages are no longer findable? 6.0 is out of support now.

```
Determining projects to restore...
/__w/runtime/runtime/jitutils/src/jit-dasm/jit-dasm.csproj : error NU1102: Unable to find package Microsoft.NETCore.App.Runtime.linux-x64 with version (= 6.0.36)
/__w/runtime/runtime/jitutils/src/jit-dasm/jit-dasm.csproj : error NU1102:   - Found 1562 version(s) in dotnet7 [ Nearest version: 7.0.0-alpha.1.21[41](https://github.com/dotnet/runtime/actions/runs/11804422106/job/32884591447?pr=109714#step:5:42)7.28 ]
/__w/runtime/runtime/jitutils/src/jit-dasm/jit-dasm.csproj : error NU1102:   - Found 174 version(s) in dotnet-public [ Nearest version: 7.0.0-preview.1.22076.8 ]
/__w/runtime/runtime/jitutils/src/jit-dasm/jit-dasm.csproj : error NU1102:   - Found 0 version(s) in dotnet-libraries
/__w/runtime/runtime/jitutils/src/jit-dasm/jit-dasm.csproj : error NU1102:   - Found 0 version(s) in myget-legacy
  Failed to restore /__w/runtime/runtime/jitutils/src/jit-dasm/jit-dasm.csproj (in 1.28 sec).
```

--- a/src/coreclr/jit/morph.cpp
+++ b/src/coreclr/jit/morph.cpp
@@ -12949,15 +12949,14 @@ bool Compiler::fgMorphBlockStmt(BasicBlock*     block,
         // We should not convert it to a ThrowBB.
         if ((block != fgFirstBB) || !fgFirstBB->HasFlag(BBF_INTERNAL))
         {
-            if (!block->KindIs(BBJ_THROW))
-            {
-                // Convert block to a throw bb
-                fgConvertBBToThrowBB(block);
+            // Convert block to a throw bb, or make it rarely run if already a throw.
+            //
+            const bool isThrow = block->KindIs(BBJ_THROW);
+            fgConvertBBToThrowBB(block);
 
-                if (invalidateDFSTreeOnFGChange)
-                {
-                    fgInvalidateDfsTree();
-                }
+            if (!isThrow && invalidateDFSTreeOnFGChange)
+            {
+                fgInvalidateDfsTree();
             }
         }
 

--- a/src/coreclr/jit/valuenum.cpp
+++ b/src/coreclr/jit/valuenum.cpp
@@ -10806,6 +10806,9 @@ PhaseStatus Compiler::fgValueNumber()
         }
     }
 
+    assert(m_dfsTree != nullptr);
+    assert(m_loops != nullptr);
+
     m_blockToLoop = BlockToNaturalLoopMap::Build(m_loops);
     // Compute the side effects of loops.
     optComputeLoopSideEffects();

